tab_news.md

```diff
@@ -6,6 +6,12 @@ order: 3
 tags: mstg
 ---
 
+## May 23rd 2020: New buildsystem and release upcoming!
+
+As already shared during the virtual Dutch Chapter meetup: we are going to release a new version of the MSTG soon. We have been quite busy with moving it to the new build-system first as the document got too large for our old tooling. This is a great incentive as well to think of how we can make the guide more focused so that the size does not matter ;-).
```

Comment:
```suggestion
As already shared during the virtual Dutch Chapter meetup: we are going to release a new version of the MSTG soon. We have been quite busy with moving it to the new build system first as the document got too large for our old tooling. This is a great incentive as well to think of how we can make the guide more focused so that the size does not matter ;-)
```